### PR TITLE
docs: add information about the onLAN

### PR DESCRIPTION
**Pull request by dsluijk**

### CODE REVIEW

File: data/faq.md [hunk 1]
```diff
@@ -1,32 +1,22 @@
...
 ### How does the event look like this year?
 
 As you can probably imagine AreaFiftyLAN will be pretty different, as safety of our participants and staff will always come first.
-A lot is still unclear, and we want to hold off with any new information till we are sure.
-What we can tell you is that the event will at least partly take place online, and if we do have physical spots they will be very limited.
-
-### When will he event take place?
-
-The event will take place from Friday the 28th of February till Sunday the 1st of March. Further details will be made available at a later date.
+With this in mind, we had to make the hard choice to organize the LAN completely online this year.
```
Comment (skamoen) [inline data/faq.md:4]: Let's keep calling it _the event_, as you're doing everywhere else in the texts

File: data/faq.md [hunk 2]
```diff
@@ -1,32 +1,22 @@
...
+Don't worry, it will still be an continous three day event packed with tournaments and casual games.
```
Comment (skamoen) [inline data/faq.md:5]: continuous

File: data/faq.md [hunk 3]
```diff
@@ -1,32 +1,22 @@
...
+To tie it all together we will also have a stream running for a large portion of the day.
```
Comment (skamoen) [inline data/faq.md:6]: _also_ implies there's more?

**Review by skamoen — CHANGES_REQUESTED** [review]

File: data/faq.md [hunk 4]
```diff
@@ -1,32 +1,22 @@
-### What is the Pre-LAN?
-
-As a fun little event we will have a smaller LAN, entirely for free!
-Here you can get a taste for the LAN for just one evening!
-The event is entirely for free, and will take place February 19th at 20:00 (CET).
-You can check out our [Discord](https://discord.gg/QefnMzNGG6) for more information!
-
 ### How does the event look like this year?
```
Comment (DevDonut) [inline data/faq.md:1]: "What does the event look like" of "How does the event look"

File: data/faq.md [hunk 5]
```diff
@@ -1,32 +1,22 @@
...
-The event will take place from Friday the 28th of February till Sunday the 1st of March. Further details will be made available at a later date.
+With this in mind, we had to make the hard choice to organize the LAN completely online this year.
+Don't worry, it will still be an continous three day event packed with tournaments and casual games.
+To tie it all together we will also have a stream running for a large portion of the day.
 
-### What happens when the tickets are sold out?
+### When will the event take place?
 
-When the tickets are sold out, the only way of getting a ticket is to be put on a waiting list. Once a ticket becomes available, you will be noted of the person trying to sell the ticket. There will be no second round of the ticket sale, so make sure you get your tickets on time!
+The event will take place from Friday the 26th of February till Sunday the 28st. Further details will be made available at a later date.
```
Comment (DevDonut) [inline data/faq.md:10]: 28th

Comment (DevDonut) [reply data/faq.md:10]: made available -> released

File: data/faq.md [hunk 6]
```diff
@@ -1,32 +1,22 @@
...
 
-### Is there a minimum age to participate?
+### I don't have any friends who want to participate, will it still be fun?
 
-No, there is not. If you are younger than 18, you should be able to present any form of authorization by caretakers/parents upon entry to the event.
+Yes! There will be plenty of opportunity to socialize with other participants, and we will try to create groups of single players.
```
Comment (DevDonut) [inline data/faq.md:14]: , and -> and

File: data/faq.md [hunk 7]
```diff
@@ -1,32 +1,22 @@
...
 
-### Can I sleep on location?
+### Are there still prizes this year?
 
-Unfortunately, due to the pandemic we won't have any sleeping spot available this year.
+Yes, of course. We will have a comparable prizes available as last year. More details will follow later. On top of that we will have a (limited amount of) goodie bags available.
```
Comment (DevDonut) [inline data/faq.md:18]: a comparable -> "a comparable amount" of "comparable"

File: data/faq.md (hunk 4 above)
Comment (dsluijk) [reply data/faq.md]: Updated it!

File: data/faq.md (hunk 1 above)
Comment (dsluijk) [reply data/faq.md]: Sure

File: data/faq.md (hunk 2 above)
Comment (dsluijk) [reply data/faq.md]: Hmm, autocorrect did not pick that one up. Changed it.

File: data/faq.md (hunk 3 above)
Comment (dsluijk) [reply data/faq.md]: It does refer to the previous line, so the "more" here refers the tournaments and casual games. It does flow a bit better without so I removed it.

File: data/faq.md (hunk 5 above)
Comment (dsluijk) [reply data/faq.md]: :+1:

File: data/faq.md (hunk 6 above)
Comment (dsluijk) [reply data/faq.md]: sure

File: data/faq.md (hunk 7 above)
Comment (dsluijk) [reply data/faq.md]: sure

**Review by dsluijk — COMMENTED** [review]
I've updated the PR following your comments, please review again @DevDonut @skamoen

**Review by DevDonut — APPROVED** [review]

**Review by skamoen — APPROVED** [review]

**Pull request merged**